lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -3562,7 +3557,7 @@ namespace Js
         newFunctionBody->m_isFromNativeCodeModule = this->m_isFromNativeCodeModule;
     }
 
-    FunctionBody *
+    ParseableFunctionInfo *
```

Comment:
It should be safe to get rid of the clone code now, replace with assert. I'll see if I have time to get rid of the caller side of the code.